components/employmentOutlook/Search.js

```diff
@@ -0,0 +1,135 @@
+import { InstantSearch, Configure, connectHits } from 'react-instantsearch-dom';
+import algoliasearch from 'algoliasearch/lite';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import Divider from '@material-ui/core/Divider';
+import firebase from 'firebase/app';
+import FormControl from '@material-ui/core/FormControl';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import Typography from '@material-ui/core/Typography';
+
+import { useAuth } from '../Auth';
+import AutocompleteDropdown from './AutocompleteDropdown';
+import CountyList from './CountyList';
+import FavorabilityDialog from './FavorabilityDialog';
+import employmentInputValidation from './EmploymentInputValidation';
+import useFormValidation from '../formValidationHook';
+
+const searchClient = algoliasearch('3XON39SKZ0', '841e3368abde3ebfd860f89ddae4d60e');
+
+function Hits(props) {
+  const { hits, show, onClose } = props;
+  if (hits.length > 0) {
+    const hit = hits[0];
+    return (
+      <FavorabilityDialog
+        favorabilityValue={hit.Descriptor}
+        occupation={hit.Occupation}
+        county={hit.county}
+        onClose={onClose}
+        show={show}
+      />
+    );
+  }
+  return (
+    <Dialog open onClose={onClose}>
+      <Typography variant="h2">No data found!</Typography>
+    </Dialog>
+  );
+}
+
+const CustomHits = connectHits(Hits);
+
+function Search() {
+  const { userDocRef } = useAuth();
+  const [searching, setSearching] = useState(false);
+
+  const initialState = {
+    occupation: '',
+    county: '',
+  };
+
+  const submit = values => {
+    setSearching(true);
+
+    const timestamp = firebase.firestore.FieldValue.serverTimestamp();
+    const employmentOutlook = {
+      occupation: values.occupation,
+      county: values.county,
+      timestamp,
+    };
+    userDocRef.set({ employmentOutlook }, { merge: true });
```

Comment:
Nice -- thanks for storing this.

Could we also push an Intercom event?

Reply:
done